### DISCUSSION TITLE
Sheet Resistance in 3D Sim

### DISCUSSION THREAD

**Discussion by alexsaadfalcon** [NanoComp/meep#2397] [Q&A]
Hello, I am trying to model a PCB represented as a substrate with infinitesimally thin conductor on top. Can MEEP handle thin materials in 3D simulations?

**Comment by smartalecH** [comment]
https://meep.readthedocs.io/en/latest/FAQ/#how-do-i-model-graphene-or-other-2d-materials-with-single-atom-thickness

**Reply by alexsaadfalcon** [reply]
Thanks for your reply. I am looking for something like a sheet impedance/conductance, which seems like this would function as with a sufficiently high resolution
